crates/extensions/az_mapper_ext/src/converter.rs

```diff
@@ -81,12 +66,22 @@ impl AzureConverter {
     fn try_convert_te_topics(
         &mut self,
         input: &MqttMessage,
+        entity: &EntityTopicId,
         channel: Channel,
     ) -> Result<Vec<MqttMessage>, ConversionError> {
+        // don't convert mosquitto bridge notification topic
+        // https://github.com/thin-edge/thin-edge.io/issues/2236
+        if entity.as_str() == MOSQUITTO_BRIDGE_TOPIC_ID {
+            return Ok(vec![]);
+        }
+
         self.size_threshold.validate(input)?;
 
         match &channel {
-            Channel::Measurement { .. } | Channel::Event { .. } | Channel::Alarm { .. } => {
+            Channel::Measurement { .. }
+            | Channel::Event { .. }
+            | Channel::Alarm { .. }
+            | Channel::Health => {
                 let payload = self.with_timestamp(input)?;
                 let output = MqttMessage::new(&self.mapper_config.out_topic, payload);
                 self.size_threshold.validate(&output)?;
```

Comment:
I hope the size validation can be hidden in one-level higher. As of now, this needs to be called everywhere a MQTT message is generated. Easy to forget to call it. The size threshold is determined by the type of cloud (c8y, az, aws..). How about adding `size_threshold.validate()` where `try_convert()` is called?

https://github.com/thin-edge/thin-edge.io/blob/d3ed45f4f1f8909a113693840bfec18b5bbc55c6/crates/extensions/az_mapper_ext/src/converter.rs#L133
```
impl Converter for AzureConverter {
    type Input = MqttMessage;
    type Output = MqttMessage;
    type Error = Infallible;

    fn convert(&mut self, input: &Self::Input) -> Result<Vec<Self::Output>, Self::Error> {
        let messages_or_err = self.try_convert(input);
        // add validation here
        Ok(self.wrap_errors(messages_or_err))
    }
}
```

Then we can remove tehe line 78, `self.size_threshold.validate(input)?;`. This is improvised way to check the size... the logic is here: the output message should be checked, but since input and output messages are almost same, if input message is too large, we presume the output is also too large.

Reply:
Did the same as i did in AWS converter: e39f632c2e